examples/proc_fieldSweep.py

```diff
@@ -0,0 +1,97 @@
+"""Check ppt value using a field sweep
+======================================
+
+A DC offset correction, initial frequency slice 
+and rough centering are applied to a previously
+acquired TEMPOL dataset. It is then sliced more based
+on the 1D diagnostic plot labeled 'Field Slicing'. The 
+field is converted to ppt using the frequency stored as
+an acquisition parameter in the dataset and the effective
+gamma used that day. This will verify the previously required
+ppt value for a given spin label.
+"""
```

Comment:
there are 2 files in the PR that conflict -- `examples/proc_fieldSweep.py` and `pro_fieldSweep.py`

Reply:
I'm confused, when I "git merge master --no-ff" it just says Already up to date. When I diff against master it shows I have proc_fieldsweep.py and examples/proc_fieldsweep.py as well as the changes in load_data which would be expected as these were not in the master branch. These are new

Reply:
you shouldn't be adding scripts to the main repo directory.

also, these two files genuinely seem to have the same purpose -- so which one am I reviewing?

Reply:
resolved in https://jmfrancklab.slack.com/archives/CV553TPRV/p1643747211231269